deploy/supervisord.conf

```diff
@@ -1,46 +0,0 @@
-; # Supervisor Config File
```

Comment:
Question: Why is this removed? I am pretty sure we are linking to this in the docs for bare metal installs

Reply:
Ah you're right. I did a cursory search and thought it was orphaned.. I'll add it back in.